jenkins2/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+jenkins2:
+  build: .
+  environment:
+    JAVA_OPTS: "\
+      -Djava.awt.headless=true \
+      -Djenkins.install.runSetupWizard=false \
+      -Dorg.apache.commons.jelly.tags.fmt.timeZone=Europe/London \
```

Comment:
This is intentional. The docker-compose will template for salt will have this config in it.